tests/testthat/_snaps/cdc_growth_charts.md

```diff
@@ -0,0 +1,60 @@
+# get_cdc_data Test 1: CDC weight-for-age chart
```

Comment:
this unit testing approach is new to me sorry. could you explain a little how it works?